ci/build_container/build_recipes/yaml-cpp.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+# Using a pinned commit rather than release version to avoid taking the Boost
+# dependency present in last release.
```

Comment:
Using the pinned commit lets you skip Boost. They switched to using C++11 for the features that Boost had immediately after 0.5.3, but haven't done a release since then (and 0.5.3 was earlier last year).

We will still need this for v2 protobuf; we'll basically convert from YAML to a JSON string via libyaml-cpp parser + rapidjson emitter, then use the normal protobuf JSON -> message conversion. There's no protobuf library support for YAML.